examples/graphql-service-field-level-caching/graphql_service_field_level_caching.md

```diff
@@ -0,0 +1,23 @@
+# GraphQL service - Field-level caching
+
+The Ballerina `graphql` module provides the capability to enable GraphQL caching, which can be applied at either the field or operation-level. To enable caching at the field-level, the `cacheConfig` field in the `graphql:ResourceConfig` annotation can be used on a resource method within a `graphql:Service`. By setting this configuration, caching is enabled for the specified GraphQL field, and it can override cache configurations set at the operation-level.
```

Comment:
```suggestion
The Ballerina `graphql` module provides the capability to enable GraphQL caching, which can be applied at either the field or operation level. To enable caching at the field level, the `cacheConfig` field in the `graphql:ResourceConfig` annotation can be used on a resource method within a `graphql:Service`. By setting this configuration, caching is enabled for the specified GraphQL field, and it can override the cache configurations set at the operation level.
```